WIP: Trimmmmmmmmming (#443)
AlexsLemonade/OpenPBTA-manuscript: content/04.discussion.md

We harnessed RNA expression data for a number of analyses, yielding important biological insights across multiple brain tumor histologies.
For example, we performed subtyping of medulloblastoma tumors, for which only 35% (43/122) had subtype information from pathology reports. <!--SAMPLECOUNT-->
We accurately subtyped medulloblastomas with pathology-based subtypes using MM2S (91%; 39/43) or MedulloClassifier (95%; 41/43) [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263] and subsequently applied the consensus of these methods to subtype all medulloblastomas.

We advanced the integrative analyses and cross-cohort comparison via a number of validated modules.
We used an expression classifier to determine whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076] and the EXTEND algorithm to determine their degree of telomerase activity using a 13-gene signature [@doi:10.1038/s41467-020-20474-9].

> Interestingly, in contrast to adult colorectal cancer and gastric adenocarcinoma, in which _TP53_ loss of function is less frequent in hypermutated tumors [@doi:10.18632/oncotarget.22783; @https://doi.org/10.1038/NATURE13480], we found that hypermutant HGGs universally displayed dysregulation of _TP53_.

Reviewer: This sentence confused me, so I rephrased to what I think it means. If it doesn't mean this, some other kind of rephrase should be done.
```suggestion
Interestingly, in contrast to adult colorectal cancer and gastric adenocarcinoma, in whose hypermutated tumors _TP53_ loss of function is less frequent [@doi:10.18632/oncotarget.22783; @https://doi.org/10.1038/NATURE13480], we found that hypermutant HGGs universally displayed dysregulation of _TP53_.
```

We also show that EXTEND scores are a robust surrogate measure for telomerase activity in pediatric brain tumors.
By assessing _TP53_ and telomerase activity prospectively from expression data, information usually only attainable with DNA sequencing and/or qPCR, we can quickly incorporate oncogenic biomarker and prognostic knowledge and expand our biological understanding of these tumors.

We identified enrichment of hallmark cancer pathways and characterized the immune cell landscape across pediatric brain tumors, demonstrating tumors in some histologies, such as schwannomas, craniopharyngiomas, and low-grade gliomas, may have a inflammatory tumor microenvironment.
Of note, we observed upregulation of IFN$\gamma$, IL-1, and IL-6, and TNF$\alpha$ in craniopharyngiomas, tumors difficult to resect due to their anatomical location and critical surrounding structures.
Neurotoxic side effects have been reported when interferon alpha immunotherapy is administered to reduce cystic craniopharyngioma tumor size and/or delay progression [@doi:10.3171/2015.2.PEDS14656; @doi:10.5348/ijcri-2013-12-419-CR-13].

> Thus, additional immune vulnerabilities, such as IL-6 inhibition and immune checkpoint blockade, have recently been proposed as therapies for cystic adamantinomatous craniopharyngiomas [@doi:10.1093/neuonc/noy035; @pubmed:34966342; @pubmed:32075140; @doi:10.1007/s00401-018-1830-2; @doi:10.3389/fonc.2019.00791] and our results support this approach.

Reviewer: ```suggestion
Thus, additional immune vulnerabilities, such as IL-6 inhibition and immune checkpoint blockade, have recently been proposed as therapies for cystic adamantinomatous craniopharyngiomas [@doi:10.1093/neuonc/noy035; @pubmed:34966342; @pubmed:32075140; @doi:10.1007/s00401-018-1830-2; @doi:10.3389/fonc.2019.00791], and our results support this approach.
```

Finally, our study reproduced the overall known poor infiltration of CD8+ T cells and general low expression of _CD274_ (PD-L1) in pediatric brain tumors, further highlighting the urgent need to identify novel therapeutic strategies for tumors unlikely to respond to immune checkpoint blockade therapy.

We note that while large-scale collaborative efforts tend to take a longer time to complete, our adoption of an open science framework for OpenPBTA substantially mitigated this concern.
By maintaining all data, analytical code, and results in public repositories, we ensured that such logistics did not hinder progress in the pediatric cancer research space.

> Indeed, OpenPBTA has rapidly become a foundational data analysis and processing layer for a number of discovery research and translational projects which will continue to add other genomic modalities and analyses, such as germline, epigenomic, single cell, mRNA splicing, imaging, and model drug response data.

Reviewer: does this sneaky cut a word?
```suggestion
Indeed, OpenPBTA has rapidly become a foundational data analysis and processing layer for a number of discovery research and translational projects which will continue to add other genomic modalities and analyses, such as germline, epigenomic, single-cell, mRNA splicing, imaging, and model drug response data.
```

For example, the RNA fusion filtering module created within OpenPBTA set the stage for development of the R package _annoFuse_ [@doi:10.1186/s12859-020-03922-7] and an R Shiny application [_shinyFuse_](http://shiny.imbei.uni-mainz.de:3838/shinyFuse/).

> Using medulloblastoma subtyping and immune deconvolution analyses performed herein, Dang and colleagues showed enrichment of monocyte and microglia-derived macrophages within the SHH subgroup which they suggest may accumulate following radiation therapy [@doi:10.1016/j.celrep.2021.108917].

Reviewer: ```suggestion
Using medulloblastoma subtyping and immune deconvolution analyses performed herein, Dang and colleagues showed that SHH tumors are enriched with monocyte and microglia-derived macrophages, which may accumulate following radiation therapy [@doi:10.1016/j.celrep.2021.108917].
```

> Expression and copy number analyses were used to demonstrate that _GPC2_ is a highly expressed and copy number gained immunotherapeutic target in ETMRs, medulloblastomas, choroid plexus carcinomas, H3 wildtype high-grade gliomas, as well as DMGs.

Reviewer: ```suggestion
Expression and copy number analyses demonstrated that _GPC2_ is a highly expressed and copy-number gained immunotherapeutic target in ETMRs, medulloblastomas, choroid plexus carcinomas, H3 wildtype high-grade gliomas, and DMGs.
```

> This led Foster and colleagues to subsequently develop a chimeric antigen receptor (CAR) directed against _GPC2_, for which they show preclinical efficacy in mouse models [@doi:10.1136/jitc-2021-004450].

Reviewer: ```suggestion
Foster and colleagues therefore developed a chimeric antigen receptor (CAR) directed against _GPC2_, which shows preclinical efficacy in mouse models [@doi:10.1136/jitc-2021-004450].
```

> Another study harnessed OpenPBTA to integrate germline variants and discovered that pediatric HGG patients whose tumors undergo alternative lengthening of telomeres have enrichment of pathogenic or likely pathogenic germline variants in genes in the MMR pathway, oncogenic _ATRX_ mutations, and increased TMB [@doi:10.1093/neuonc/noac278].

Reviewer: This sentence read oddly to me.. please make sure that my suggestion keeps the meaning the same! But, I think everything needed a verb?
 
```suggestion
Another study harnessed OpenPBTA to integrate germline variants, discovering that pediatric HGG patients whose tumors undergo alternative lengthening of telomeres are enriched for pathogenic or likely pathogenic germline variants in MMR pathway, possess oncogenic _ATRX_ mutations, and have increased TMB [@doi:10.1093/neuonc/noac278].
```

> Moreover, OpenPBTA has enabled a framework to support real-time integration of clinical trial subjects as they enrolled on the PNOC008 high-grade glioma clinical trial [@clinicaltrials:NCT03739372] or PNOC027 medulloblastoma clinical trial [@clinicaltrials:NCT05057702], allowing researchers and clinicians to link tumor biology to translational impact through clinical decision support during tumor board discussions.

Reviewer: You could probably keep only one of "**translational impact** through **clinical decision**" here

Author: going to keep both for now

> Finally, as part of the the NCI's Childhood Cancer Data Initiative (CCDI), the OpenPBTA project was recently expanded into a pan-pediatric cancer effort ([https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis)) which enabled creation of the pediatric Molecular Targets Platform ([https://moleculartargets.ccdi.cancer.gov/](https://moleculartargets.ccdi.cancer.gov/)) in support of the RACE Act.

Reviewer: ```suggestion
Finally, as part of the the NCI's Childhood Cancer Data Initiative (CCDI), the OpenPBTA project was recently expanded into a pan-pediatric cancer effort ([https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis)) which enabled the pediatric Molecular Targets Platform ([https://moleculartargets.ccdi.cancer.gov/](https://moleculartargets.ccdi.cancer.gov/)) in support of the RACE Act.
```

An additional, large-scale cohort of >1,500 tumor samples and associated germline DNA is undergoing harmonization as part of CBTN CCDI-Kids First NCI and Common Fund project ([https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick](https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick)).

> Like the original OpenPBTA cohort, data will be processed and released in near real-time via the Kids First Data Resource and integrated with OpenPBTA data.

Reviewer: > and integrated with OpenPBTA data.

This phrase reads to me like we will keep updating OpenPBTA data releases, as opposed to the other way around?

Author: this was an adam add ;) - let me think how to rephrase

Author: check [2917b01](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/443/commits/2917b01b577ffba399139f6722e04f4942de8fa3)
